pkg/image/build.go

```diff
@@ -20,13 +20,19 @@ import (
 
 const dockerignoreBackupPath = ".dockerignore.cog.bak"
 const weightsManifestPath = ".cog/cache/weights_manifest.json"
+const bundledSchemaFile = ".cog/openapi_schema.json"
+const bundledSchemaPy = ".cog/openapi_schema.py"
```

Comment:
```suggestion
const bundledSchemaPy = ".cog/schema.py"
```

It's not an OpenAPI schema.